Cargo.toml

```diff
@@ -3,7 +3,8 @@ authors = ["The Rust Project Developers"]
 description = "A collection of numeric types and traits for Rust, including bigint,\ncomplex, rational, range iterators, generic integers, and more!\n"
 documentation = "http://rust-num.github.io/num"
 homepage = "https://github.com/rust-num/num"
-keywords = ["mathematics", "numerics"]
+keywords = ["mathematics", "numerics", "bignum"]
+categories = [ "algorithms", "data-structures", "science", "cryptography" ]
```

Comment:
It seems a little bit of a stretch to say "science", but there aren't any math categories, so I guess that will do.

I'm less comfortable with "cryptography" though.  Yes, bigints are usually needed to implement crypto, but we don't implement any crypto algorithms here.